bazelisk_test.sh

```diff
@@ -214,6 +252,18 @@ echo "# test_bazel_version_from_environment"
 test_bazel_version_from_environment
 echo
 
+echo "# test_bazel_version_prefer_environment_to_bazeliskrc"
+test_bazel_version_prefer_environment_to_bazeliskrc
+echo
+
+echo "# test_bazel_version_from_bazeliskrc"
+test_bazel_version_from_bazeliskrc
+echo
+
+echo "# test_bazel_version_prefer_bazeliskrc_to_bazelversion_file"
+test_bazel_version_prefer_bazeliskrc_to_bazelversion_file
+echo
+
```

Comment:
You'll have to move these below into the `if [[ $BAZELISK_VERSION == "GO" ]]; then` block, because the bazeliskrc feature isn't implemented in the Python version. :)